portfolio.html

```diff
@@ -0,0 +1,120 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link href="portstyle.css" type="text/css" rel="stylesheet">
+  
+</head>
+<body>
+
+	<section class="navigationBar">
+		<div class="navigationButtons">
+			<div class="home">
+			<a href="#">Home</a>
+		</div>
+			<div class="about">
+			<a href="#">About</a>
+		</div>
+			<div class="portfolio">
+			<a href="#">Portfolio</a>
+		</div>
+			<div class="socials">
+			<a href="#">Socials</a>
+		</div>
+		</div>
+		
+	</section>
+	
+	<section class="mainImage">
+		<div class="backgroundImage">
+		<div class="textoverlayed">
+		<div class="overlayText">
```

Comment:
I think you can achieve the same affect you are currently getting by using flex box.
If you delete the textoverlayed div and it's css and add the following css to your background image css:
display: flex;
justify-content: center;
align-items; center;

These three properties will center this element within your background image.

Then just remove:

position: fixed;
left: 42.5%; 
top: 280px;
width: 14%;

and you will have a centered element that will respond to browser size changes. Less coding later!

Reply:
Josh mate that was a bloody fantastic shout. Have made the amendments.